dotCMS/src/main/java/com/dotcms/graphql/datafetcher/RelationshipFieldDataFetcher.java

```diff
@@ -23,7 +23,13 @@
     @Override
     public Object get(final DataFetchingEnvironment environment) throws Exception {
         try {
-            final Contentlet contentlet = environment.getSource();
+            final Contentlet nonCachedContentlet = environment.getSource();
+
+            // let's use the cache content so relationships are also cached
+            final Contentlet contentlet = APILocator.getContentletAPI()
```

Comment:
Should we use the  findContentletByIdentifierOrFallback  method here?

Reply:
I think findContentletByIdentifier is ok here